### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by anderslindho**
Minor bugfixes, primarily refactoring of e3 wrapper to support usage according to naming convention as well as to support systems not following standard module configurations. See example startup script for usage.

### CODE REVIEW

File: iocsh/evr300.iocsh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#- ###   ESS MTCA MRF EVR iocsh                     ###
```
Comment (nicklasholmberg2) [inline iocsh/evr300.iocsh:1]: Why was this file edited?

File: template/README.md [hunk 2]
```diff
@@ -121,24 +118,24 @@ can be translated into
...
+           { V		    , 1	      	, 2	}
+           { V	    	, 2		      , 2	}
+           { V  	  	, 3	      	, 2	}
+   	       { FANSPEED	, 1	      	, 4	}
+           { FANSPEED	, 2	      	, 4	}
+           { FANSPEED	, 3	      	, 4	}
+           { FANSPEED	, 4	      	, 4	}
+           { FANSPEED	, 5	      	, 4	}
+           { FANSPEED	, 6	      	, 4	}
+}
```
Comment (nicklasholmberg2) [inline template/README.md:133]: White spaces

File: template/mch_type1_ess.substitutions [hunk 3]
```diff
@@ -84,70 +84,44 @@
...
-      	   { MCH	, 3	}
-	   { CLK        , 60    }
-	   { HUB	, 61	}
-}	   
+pattern { id,       fruid   }
+        { MCHCM,    0       }
+        { MCH,      3	    }
+        { CLK,      60      }
+        { HUB,      61      }
+}
```
Comment (nicklasholmberg2) [inline template/mch_type1_ess.substitutions]: why whitespaces before last bracket?

File: template/ifc1410_ess.substitutions [hunk 4]
```diff
@@ -31,42 +31,23 @@ global {dev=$(_PREF), link=$(_SESS), id=$(_ID), fruid=$(_FRUID)}
 
 file "fru_common_ess.template"
 {
-   pattern {  unit    } 
-   	   { $(_UNIT)_  }
+pattern { unit      } 
+   	    { $(_UNIT)  }
```
Comment (nicklasholmberg2) [inline template/ifc1410_ess.substitutions]: white spaces

File: template/mch_type1_ess.substitutions [hunk 5]
```diff
@@ -84,70 +84,44 @@
...
 {
-   pattern { id		, fruid	}
-           { MCHCM	, 0	}
-      	   { MCH	, 3	}
-	   { CLK        , 60    }
-	   { HUB	, 61	}
-}	   
+pattern { id,       fruid   }
+        { MCHCM,    0       }
+        { MCH,      3	    }
```
Comment (nicklasholmberg2) [inline template/mch_type1_ess.substitutions]: last bracket

File: template/mtca9u-crate-ess.substitutions [hunk 6]
```diff
@@ -4,64 +4,39 @@
...
-# * 1 MTCA EVR 300U (AMC2, FRUID 6)
-#
-# The total number of PVs : 461
 
 global {dev=$(dev), link=$(link), location=$(location)}
 
 file "system_common_ess.template"
 {
-   pattern {} {} 
+pattern {} {}
```
Comment (nicklasholmberg2) [inline template/mtca9u-crate-ess.substitutions]: Pattern correct?
```
pattern {}
        {}
```

File: template/mtca9u-crate-ess.substitutions [hunk 7]
```diff
@@ -4,64 +4,39 @@
...
-{
-   pattern { unit	, fruid }
-           { 1_  	, 5	}
-}
-
-file "mtcaevr300_ess.db"
-{
-   pattern { unit	, fruid }
-           { 1_  	, 6	}
+pattern {} {}
```
Comment (nicklasholmberg2) [inline template/mtca9u-crate-ess.substitutions]: pattern?

File: template/mtca9u-crate-ess.substitutions [hunk 8]
```diff
@@ -4,64 +4,39 @@
...
 }
 
-
-# Still no information from others
-# Tuesday, January 22 12:39:18 CET 2019
-
 file "others_type1_ess.db"
 {
-   pattern {} {}
+pattern {} {}
```
Comment (nicklasholmberg2) [inline template/mtca9u-crate-ess.substitutions:41]: pattern?

**Review by nicklasholmberg2 — COMMENTED** [review]
Added some comments on white spaces

**Review by nicklasholmberg2 — CHANGES_REQUESTED** [review]
Address white spaces formatting

File: template/README.md (hunk 2 above)
Comment (anderslindho) [reply template/README.md:133]: markdown file, irrelevant formatting

**Pull request merged**